### ISSUE TITLE
Missing data value

### ISSUE THREAD

**Issue by louwrentius** [tomek-l/pms7003#2]
Hello, 

If I read the datasheet correctly, I notice 12 values that are returned, but the library only returns 11. 

It seems that you are missing the N2_5 value. Below is the fixed list as I see it.

My apologies if I'm mistaken.

```
BYTES_MEANING = {
    1 :'pm1_0cf1',
    2 : 'pm2_5cf1',
    3 : 'pm10cf1',
    4 : 'pm1_0',
    5 : 'pm2_5',
    6 : 'pm10',
    7 : 'n0_3',
    8 : 'n0_5',
    9 : 'n1_0',
   10 : 'n2_5',
    11 : 'n5_0',
    12 : 'n10',
 }
```

I also compared it with the list on this blog.
https://joshefin.xyz/air-quality-with-raspberrypi-pms7003-and-java/

**Comment by tomasz-lewicki** [comment]
You're absolutely right. Thanks for pointing this out!

**Comment by tomasz-lewicki** [comment]
Fixed in 2095d9f6c66dbe33a9d060017394a9a90936c451
